src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java

```diff
@@ -169,10 +169,11 @@ public void collect(Collector collector) {
         LOG.info("GitHubCollectorTask:collect start, total enabledRepos=" + enabledRepos.size());
         for (GitHubRepo repo : enabledRepos) {
             repoCount++;
-            String repoUrl = repo.getRepoUrl() + "/tree/" + repo.getBranch();
+            String repoUrl = repo==null?"null":(repo.getRepoUrl() + "/tree/" + repo.getBranch());
```

Comment:
Can we use StringUtils.isEmpty in place of the null check?

Reply:
See the response below